app/__init__.py

```diff
@@ -47,10 +48,16 @@ def __call__(self, environ, start_response):
         return self.app(environ, start_response)
 
 
-def create_app():  # noqa: C901  pylint: disable=too-complex
+def create_app(setting_overrides=None):  # noqa: C901  pylint: disable=too-complex
     application = Flask(__name__, static_url_path='/s', static_folder='../static')
     application.config.from_object(settings)
 
+    secrets = yaml.safe_load(open(os.getenv('EQ_SECRETS')))
```

Comment:
I initially had this in application.py as i thought it would then make testing easier but @ajmaddaford pointed out that it would be better if all of the loading of application config was in the same place.